pyam/iiasa.py

```diff
@@ -35,13 +36,32 @@
 You are connected to the {} scenario explorer hosted by IIASA.
  If you use this data in any published format, please cite the
  data as provided in the explorer guidelines: {}
-""".replace("\n", "")
+""".replace(
+    "\n", ""
+)
 IXMP4_LOGIN = "Please run `ixmp4 login <username>` in a console"
 
 # path to local configuration settings
 DEFAULT_IIASA_CREDS = Path("~").expanduser() / ".local" / "pyam" / "iiasa.yaml"
 
 
+def list_platforms():
+    """Print all available ixmp4 platforms hosted by IIASA"""
+
+    platforms = ixmp4.conf.settings.manager.list_platforms()
+    utils.echo("IIASA platform".ljust(20) + "Access".ljust(10) + "Notice\n")
```

Comment:
Is there a specific reason to use the ixmp functions for displaying information other than the colors?

Reply:
Alright, fine with me just wanted to know if I was missing something